packages/featureflags/featureToggleManager.js

```diff
@@ -53,11 +53,14 @@ class FeatureToggleManager {
         }
     }
 
-    async isFeatureEnabled (keystoneContext, featureName, featuresContext) {
-        const context = get(keystoneContext, 'req', null)
+    async #getContext (keystoneContext) {
+        return get(keystoneContext, 'req', null)
             ? keystoneContext
             : { req: { features: await this.fetchFeatures() } }
+    }
 
+    async isFeatureEnabled (keystoneContext, featureName, featuresContext) {
+        const context = this.#getContext(keystoneContext)
```

Comment:
You change `getContext` method to async. So you need to add await here
```suggestion
        const context = await this.#getContext(keystoneContext)
```